docs/release-notes/highlights-7.7.0.asciidoc

```diff
@@ -90,6 +90,17 @@ The distance filter is in addition to the tools for filtering data by custom sha
 [role="screenshot"]
 image:release-notes/images/7-7-maps-distance-filter.png[Distance filters in Elastic Maps]
 
+[float]
+[[file-uploader-highlights]]
+=== {data-viz} now recommends {filebeat} config
+
+experimental[] If you upload a file to the *{data-viz}*, it now has the ability to recommend a {filebeat} 
+configuration. You can then use {filebeat} to add more data of 
+the same structure to the same index that was created during an initial upload.
+
+[role="screenshot"]
```

Comment:
Since the side navigation is constantly changing, we don't include it in our screen captures.